proto/messages.proto

```diff
@@ -215,7 +225,6 @@ message Asset {
 //  optional Preview preview = 2;  // deprecated - preview was completely replaced
   oneof status {
     NotUploaded not_uploaded = 3;
-    RemoteData uploaded = 4;
```

Comment:
Not sure about the reason why this must go now.

Reply:
Definitely removed by accident

Reply:
Yep

Reply:
Fixed